src/Components/Availability/useAvailability.ts

```diff
@@ -77,7 +77,7 @@ export function useAvailability(category: Models.ICategory, topic: string) {
       const rs = DB.RegionService.find(x => x.Id === service.id);
 
       if (!rs) {
-        log.warn("Service not found.", service);
+        log.info("Service not found.", service);
```

Comment:
Shouldn't that be a warning?

Reply:
because it's a hidden service by design if not found.

Reply:
![image](https://github.com/user-attachments/assets/338f3ab4-cdd6-4712-8eb3-3794fffff143)
![image](https://github.com/user-attachments/assets/6b8da840-e1c3-42c1-97df-3d51aacf7101)